libs/pbd/fpu.cc

```diff
@@ -27,10 +27,6 @@
 #include <stdint.h>
 #include <assert.h>
 
-#ifdef PLATFORM_WINDOWS
-#include <intrin.h>
-#endif
-
```

Comment:
Best to leave includes at the top.
See item 2 at https://ardour.org/styleguide.html